server/controllers/user.controllers.js

```diff
@@ -93,6 +94,18 @@ const UserController = {
       })
       .catch((error) => next(error));
   },
+
+  getAllRegisteredUsers(req, res, next) {
+    getAllRegisteredUsers()
+      .then((users) => {
+        if (users.length > 0) {
+          res.status(httpStatus.OK).json({ success: true, users });
+        } else {
+          res.status(httpStatus.OK).json({ success: false, message: 'No users registered', users });
+        }
```

Comment:
Personally, I dont know why you like separating this. I believe you could just collapse everything into a single line.

Nit: You had a typo on line 104, success should be true.
```suggestion
      res.status(httpStatus.OK).json({ success: true, users });
```